app/commands/scrape_command.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class ScrapeCommand
+  attr_reader :keyword
+
+  def initialize(keyword_id:)
+    @keyword_id = keyword_id
+  end
+
+  # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/AbcSize
+  def run
+    keyword = Keyword.find(@keyword_id)
+    html = Google::SearchService.search!(keyword.content)
+    result = Google::ParseService.new(html).call
+    persist_scrape_result keyword, result
```

Comment:
Prefer to add a lin break between variable statements and method calls for readability:

```suggestion

    persist_scrape_result keyword, result
```

Make the code breathe 🌬️

Reply:
Fixed in https://github.com/Goose97/google-scraper-ruby/pull/20/commits/defa7f47009b8c127be165fef468c28c5b83d6d9